modules/ROOT/pages/backup-restore/modes.adoc

```diff
@@ -31,21 +32,29 @@ Doing full backup...
 ...
 ----
 ====
+
+The `neo4j-admin database backup` command with the `--type=full` argument creates a xref:backup-restore/online-backup.adoc#backup-artifact[full backup artifact] file in the format of _<databasename><timestamp>.backup_, called  in the target location (`--to-path`).
+
+The `neo4j-admin database backup` command allows you to include metadata, enabling the backup to store the role-based access control (RBAC) settings associated with the database.
+
 For more information about online backup options and how to control memory usage, see xref:backup-restore/online-backup.adoc[Back up an online database].
 
-[NOTE]
-====
-A full online database backup creates a xref:backup-restore/online-backup.adoc#backup-artifact[full backup artifact] in the target location (`--to-path`).
-====
+
+=== `neo4j-admin database dump`
+
+The `neo4j-admin database dump` command creates a single-file archive, that follows the format _<databasename><timestamp>.dump_, and stores it in the xref:configuration/file-locations.adoc#data[_<NEO4J_HOME>/data_] directory.
+
+Note that when using `neo4j-admin database dump`, only the database contents are backed up; database metadata and associated RBAC settings are not included.
 
 For more information about performing a full backup against an *offline* database, see xref:backup-restore/offline-backup.adoc[Back up an offline database].
 
 
+[role=enterprise-edition]
 [[differential-backup]]
 == Differential backup
 
-
-In the online backup version, after the initial full backup, the subsequent backups attempt to use the differential mode, where just the delta of the transaction logs since the last backup are transferred and used to create a differential backup artifact (stored in the target location). Those differential backup artifacts form a xref:backup-restore/online-backup.adoc#backup-chain[backup chain].
+In the online backup version, after the initial full backup, the subsequent backups attempt to use the differential mode, where just the delta of the transaction logs since the last backup are transferred and used to create a differential backup artifact (stored in the target location).
```

Comment:
```suggestion
In the online backup version, after the initial full backup, subsequent backups attempt to use differential mode, where only the delta of the transaction logs since the last backup is transferred and used to create a differential backup artifact (stored in the target location).
```